gensim/scripts/make_wikicorpus.py

```diff
@@ -14,11 +14,13 @@
 
 This actually creates three files:
```

Comment:
Now and before it creates more than 3 files.

Reply:
nice catch, fixed